stream-chat-android-offline/detekt-baseline.xml

```diff
@@ -165,7 +165,6 @@
     <ID>MaxLineLength:UploadAttachmentsNetworkType.kt$UploadAttachmentsNetworkType$* An enumeration of various network types used as a constraint in [io.getstream.chat.android.offline.internal.message.attachments.UploadAttachmentsAndroidWorker].</ID>
     <ID>MaxLineLength:UserEntity.kt$UserEntity$*</ID>
     <ID>MaxLineLength:WhenEnrichChannel.kt$WhenEnrichChannel$fun</ID>
-    <ID>MaxLineLength:WhenUpdateLastMessage.kt$WhenUpdateLastMessage$fun</ID>
```

Comment:
any idea why this was dropped?

Reply:
No, i ran detektBaseline, could be that the issue was resolved but detektBaseline wasn't run?

Reply:
yup most likely!